src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_stacks/adf_default_pipeline.py

```diff
@@ -44,6 +44,10 @@ def generate_adf_default_pipeline(scope: core.Stack, stack_input):
     if "github" in _source_name:
         adf_github.GitHub.create_webhook_when_required(scope, _pipeline.cfn, stack_input["input"])
 
+    pipeline_triggers = stack_input["input"].get("triggers", {}).get("triggered_by", None)
```

Comment:
None is the default 😄 
```suggestion
    pipeline_triggers = stack_input["input"].get("triggers", {}).get("triggered_by")
```